cpp/src/components/connectivity.cu

```diff
@@ -135,7 +131,7 @@ std::enable_if_t<std::is_signed<IndexT>::value>
                <<"\n";
 #endif
       
-      CUGRAPH_EXPECTS( is_symmetric, "Invalid API parameter");
+      CUGRAPH_EXPECTS( is_symmetric, "Invalid API parameter; Graph must be symmetric");
```

Comment:
Semi-colon (;) after parameter looks like a typo for a colon (:)

Reply:
fixed